### ISSUE TITLE
[Feature request] Add 60 fps option for RAW video

### ISSUE THREAD

**Issue by epicfilmmaker93** [mirsadm/motioncam#40]
While incredibly performance hungry, this is already supported on some devices, so it could be great to have this feature

**Comment by nocantsin** [comment]
It would indeed be great to also have custom lower frame rates like 16fps and 18fps (which in the past were common for silent movies and small gauge/8mm film) .

**Comment by 2blackbar** [comment]
I agree, ideally some sort of setting to input our desired fps ourselves from bulb till 60 maybe even more if possible in lower resolutions.But these shouldnt be top priority, most popular fps speeds are already available.
